File: app/src/release/java/in/nerd_is/android_showcase/DebugOnly.java
package in.nerd_is.android_showcase;

import android.content.Context;

import com.facebook.stetho.Stetho;

/**
 * @author Xuqiang ZHENG on 2017/3/14.
 */
public class DebugOnly {
    public static void init(Context context) {
        // empty
    }

    public static OkHttpClient.Builder addStethoInterceptor(OkHttpClient.Builder builder) {
        // do nothing
        return builder;
    }
}

File: app/src/release/kotlin/in/nerd_is/android_showcase/DebugOnly.kt
package `in`.nerd_is.android_showcase

import `in`.nerd_is.android_showcase.DebugOnly.addStethoInterceptor
import android.content.Context

import com.facebook.stetho.Stetho
import okhttp3.OkHttpClient

/**
 * @author Xuqiang ZHENG on 2017/3/14.
 */
object DebugOnly {
    @JvmStatic fun init(context: Context) {
        // empty
    }

    fun OkHttpClient.Builder.addStethoInterceptor(): OkHttpClient.Builder {
        // do nothing
        return this
    }

    fun OkHttpClient.Builder.addLoggingInterceptor(): OkHttpClient.Builder {
        // do nothing
        return this
    }
}
